apps/google-analytics-4/frontend/src/locations/Sidebar.tsx

```diff
@@ -1,20 +1,22 @@
 import AnalyticsApp from 'components/main-app/analytics-app/AnalyticsApp';
 import { useSDK } from '@contentful/react-apps-toolkit';
-
-// TO DO: Add propertyId and reportSlug logic
+import { useApi } from 'hooks/useApi';
 
 const Sidebar = () => {
   const sdk = useSDK<any>();
```

Comment:
You can use the `SidebarExtentionSDK` type here instead of any (imported from `import { SidebarExtensionSDK } from '@contentful/app-sdk';`)

Reply:
oh thank you for this suggestion 👍 have adjusted!